extra/docs_landing.html

```diff
@@ -48,335 +48,195 @@
-      </dd>
-    </dl>
-  </details>
-
-  <h2>Get started</h2>
-  <p>New to Elastic? Learn how you can make the most of your data with the Elastic Stack.
-    Get hands-on with a solution and quickly see data in action, or start from a blank page. </p>
-  <div class="row my-4">
-    <div class="col-md-4 col-12 mb-2">
-      <a class="no-text-decoration" href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
-        <div class="card h-100">
-          <h4 class="mt-3">
-            <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt11200907c1c033aa/634d9da119d8652169cf9b2b/enterprise-search-logo-color-32px.png');"></span>
-            Search
-          </h4>
-          <p>Build custom applications with your data using Elasticsearch.</p>
-        </div>
-      </a>
+  <div role="tabpanel" id="solution-tab-contents" aria-labelledby="solution-tab" tabindex="0">
+    <div class="row">
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/elasticsearch/reference/current/index.html">
+          <div class="card h-100">
+            <h3 class="mt-3">
+              <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt11200907c1c033aa/634d9da119d8652169cf9b2b/enterprise-search-logo-color-32px.png');"></span>
+              Search
+            </h3>
+            <p>Build custom applications with your data using Elasticsearch.</p>
+          </div>
+        </a>
+      </div>
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/observability/current/index.html">
+          <div class="card h-100">
+            <h3 class="mt-3">
+              <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltaa08b370a00bbecc/634d9da14e565f1cdce27f7c/observability-logo-color-32px.png');">
+              </span>
+              Observability
+            </h3>
+            <p>Monitor hosts by collecting logs and metrics with Elastic Observability.</p>
+          </div>
+        </a>
+      </div>
+      <div class="col-md-4 col-12 mb-2">
+        <a class="no-text-decoration" href="en/security/current/es-overview.html">
+          <div class="card h-100">
+            <h3 class="mt-3">
+              <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt5e0e0ad9a13e6b8c/634d9da18473831f96bbdf1e/security-logo-color-32px.png');">
+              </span>
+              Security
+            </h3>
+            <p>Detect, investigate, and respond to threats with Elastic Security for SIEM.</p>
```

Comment:
SIEM is only one of the three main use cases for Elastic Security (SIEM, cloud, and endpoint). Could we just omit? (This is what we did for the equivalent [serverless landing page](https://docs.elastic.co/serverless).)
```suggestion
            <p>Detect, investigate, and respond to threats with Elastic Security.</p>
```

<img width="999" alt="image" src="https://github.com/elastic/docs/assets/87339667/b8c81d1d-b1e4-43dc-9c98-abbdc6cc77f2">